dataflow/manual/content.tex

```diff
@@ -1782,6 +1782,31 @@ \section{Example: Very Busy Expression Analysis}
 
 \flow{BusyExprSimple}{.33}{1}{Very Busy Expression Analysis illustration. Intermediate analysis results are shown.}
 
+\section{Example: Reaching Definitions}
+
+The reaching definitions for a given program point are those assignments that
+may have updated the current values of variables.
+This reaching definitions analysis is a standard example of a forward analysis.
+
+\textbf{The node.} A class \code{ReachingDefinitionsNode} is used as an
+abstract value, which can only wrap \code{AssignmentNode}. The reaching definitions
+processes such values in the store.
+
+\textbf{The store.} The reaching definitions store \code{ReachingDefinitionsStore} has a field
+\code{reachingDefSet} which contains a set of \code{ReachingDefinitionsNode}s. The store defines methods
+\code{putDef(ReachingDefinitionValue)} and \code{killDef(Node)} to add
+and kill reaching definitions.
+
+\textbf{The transfer function.} The transfer function visits \code{AssignmentNode} to
+update the information of reaching definitions in the stores. A reaching definition will
+be killed in the store when its lhs is same as that of the new generated value, and the new
```

Comment:
I think this document doesn't use lhs/rhs abbreviations. Can you check and if necessary expand.

Reply:
Thanks, yes, now it is expanded in my local.